core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java

```diff
@@ -71,6 +71,16 @@ protected ToElementTuple getInstance() {
         return ToElementTuple.class;
     }
 
+    @Override
+    protected Class[] getExpectedSignatureInputClasses() {
+        return new Class[]{Element.class};
+    }
+
+    @Override
+    protected Class[] getExpectedSignatureOutputClasses() {
+        return new Class[]{};
```

Comment:
I would have expected this to be an ElementTuple

Reply:
Now fixed